routes/artifacts.js

```diff
@@ -344,11 +345,18 @@ api.declare({
 /** Reply to an artifact request using taskId, runId, name and context */
 var replyWithArtifact = async function(taskId, runId, name, req, res) {
   // Load artifact meta-data from table storage
-  var artifact = await this.Artifact.load({
-    taskId:     taskId,
-    runId:      runId,
-    name:       name
-  }, true);
+  var promiseResponse = await Promise.all([
+      this.Artifact.load({
+        taskId:     taskId,
+        runId:      runId,
+        name:       name
+      }, true),
+      this.Task.load({
+        taskId: taskId
+      }, true),
+  ]);
+  var artifact = promiseResponse[0];
+  var task = promiseResponse[1];
```

Comment:
I don't like this at all...
The pattern should be:
```js
var [task, artifact] = await Promise.all([
  ...
]);
```
And even then I don't like it... Because we shouldn't load the task definition here... Why?

I know you are doing a temporary hack... So I'll allow it for now... Please label this with a comment `// TEMPORARY HACK: Remove this.Task.load(...) in the future!`

Otherwise, we might forget to remove `Task.load` when we clean up...